pkgs/development/tools/rubocop/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, lib, bundlerEnv, ruby, makeWrapper }:
+
+bundlerEnv rec {
+  name = "rubocop-${version}";
+  version = "0.47.0";
```

Comment:
Instead of this it's also possible to use `pname = "rubocop";`. This will automatically extract the version from the `gemset.nix` and hide any bin that are not part of the main "rubocop" package.